CMakeLists.txt

```diff
@@ -32,8 +32,13 @@ if(HIPIFY_INCLUDE_IN_HIP_SDK)
     message(FATAL_ERROR "HIPIFY_INCLUDE_IN_HIP_SDK is not targeting Visual Studio")
   endif()
 else()
-  find_package(Clang REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
-  find_package(LLVM REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
+  if (CUSTOM_CLANG_PATH)
+    find_package(Clang REQUIRED CONFIG PATHS ${CUSTOM_CLANG_PATH} NO_DEFAULT_PATH)
+    message(STATUS "Using ClangConfig.cmake in: ${CUSTOM_CLANG_PATH}")
+  else()
+    find_package(Clang REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
+    find_package(LLVM REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
+  endif()
```

Comment:
`CMAKE_PREFIX_PATH` is already used for specifying `custom` LLVM/clang. What are the reasons for introducing `CUSTOM_CLANG_PATH` for a separate search for the `Clang` package only?

Reply:
It will automatically pick up llvm.cmake configuration when you specify the path to clang's one, but yeah, it was not clear to me from the doc that `CMAKE_PREFIX_PATH` is being used for that purposes only. If that is the case, I will drop this. Thanks :)

Reply:
I'll make custom LLVM related text more clear in the README.md. Thanks.

Reply:
Nice! Thanks!